### PR TITLE
Distributed press playbook

### PR DESCRIPTION
**Pull request by ASoTNetworks**

### CODE REVIEW

File: distributed-press/srv1.distributed.press/roles/install-api/tasks/install-api.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+- name: Create Data directory if doesn't exists
+  file:
+    path: /home/compost/api.distributed.press/data
+    owner: compost
+    group: users
+    state: directory
+    mode: '0700'
+
+- name: Check if API db.json exists
```
Comment (benhylau) [inline distributed-press/srv1.distributed.press/roles/install-api/tasks/install-api.yml]:
```suggestion
- name: Check if API config.json exists
```

File: distributed-press/srv1.distributed.press/roles/install-api/tasks/install-api.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  stat:
+    path: /home/compost/api.distributed.press/data/config.json
+  register: config_exists
+
+- name: Copy config.json /home/compost/api.distributed.press/
+  template:
+    src: config.json
```
Comment (benhylau) [inline distributed-press/srv1.distributed.press/roles/install-api/tasks/install-api.yml:44]: So this is from the encrypted blob?

File: distributed-press/srv1.distributed.press/roles/install-hypercore/vars/main.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (benhylau) [inline distributed-press/srv1.distributed.press/roles/install-hypercore/vars/main.yml:1]: How are these generated?

**Review by benhylau — APPROVED** [review]

File: distributed-press/srv1.distributed.press/roles/install-api/tasks/install-api.yml (hunk 2 above)
Comment (ASoTNetworks) [reply distributed-press/srv1.distributed.press/roles/install-api/tasks/install-api.yml:44]: Yup it get it from https://github.com/hyphacoop/ansibles/blob/distributed-press/distributed-press/srv1.distributed.press/roles/install-api/templates/config.json

File: distributed-press/srv1.distributed.press/roles/install-hypercore/vars/main.yml (hunk 3 above)
Comment (ASoTNetworks) [reply distributed-press/srv1.distributed.press/roles/install-hypercore/vars/main.yml:1]: This is the static dat-store user and pass that I put in our password manager and it is encrypted with Ansible vault.

**Comment by benhylau** [comment]
Pls go ahead and merge :)

**Pull request merged**